Bottom: 07480b969d7b0ad95a04e5d9bd4e70fd1e87a210
Top:    260ba81cacfe7821911e950c95af6e2ef252590c
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-02-26 19:41:48 +1300

Refresh of stream-summary

---

diff --git a/testtools/testresult/real.py b/testtools/testresult/real.py
index f449c9b..1e8f097 100644
--- a/testtools/testresult/real.py
+++ b/testtools/testresult/real.py
@@ -508,7 +508,7 @@ def test_dict_to_case(test_dict):
         from testtools.testcase import PlaceHolder
     outcome = _status_map[test_dict['status']]
     return PlaceHolder(test_dict['id'], outcome=outcome,
-        details=test_dict['details'])
+        details=test_dict['details'], tags=test_dict['tags'])
 
 
 class StreamSummary(StreamToDict):
